cassandane/Cassandane/Cyrus/Notify.pm

```diff
@@ -277,6 +277,14 @@ sub test_message
     # Should get no unsolicited responses
     $res = $store->idle_response({}, 1);
     $self->assert(!$res, "no unsolicited responses");
+
+    # we enabled NOTIFY, so we should see it in client behaviors
+    if ($self->{instance}->{have_syslog_replacement}) {
+        my (@lines) = grep { /session ended/ && /notify=<1>/ }
+                      $self->{instance}->getsyslog();
+
+        $self->assert_num_gte(1, scalar @lines);
+    }
```

Comment:
Same thing as the other failing test -- we enabled NOTIFY on the client `$talk`, and that client is still connected, so we haven't logged its "session ended" yet